listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -163,13 +163,13 @@ export default class ListenCard extends React.Component<
           <MediaQuery minWidth={768}>
             <div className="col-xs-9">
               <div className="track-details">
-                <p title={listen.track_metadata.track_name}>
+                <p title={listen.track_metadata?.track_name}>
                   {getTrackLink(listen)}
```

Comment:
does the call to `_.get` in getTrackLink also deal with this situation correctly?

Reply:
That is correct.